### PR TITLE
Release automation - Winget, Chocolatey

### PR DESCRIPTION
**Pull request by abu-usama**
- Created the winget release workflow.
- Included `windows_arm64` in the supported architectures.
- Created the choco release workflow.

### CODE REVIEW

**Comment by UmanShahzad** [comment]
Do we have the token?

File: chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿$ErrorActionPreference = 'Stop';
+ 
+$packageName = 'ipinfo'
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+    packageName    = $packageName
+    url64          = 'https://github.com/ipinfo/cli/releases/download/ipinfo-3.3.0/ipinfo_3.3.0_windows_amd64.zip'
+    checksum64     = '2ab00f6a289d308b9ac04be9153aa566e93acff3da8a62dd283022a0cb6ee34d'
```
Comment (UmanShahzad) [inline chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: Will this be auto-updated?

Comment (abu-usama) [reply chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: yes

Comment (UmanShahzad) [reply chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: You mean in this repo itself? How? The app makes a commit or smth?

Comment (UmanShahzad) [reply chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: Elaborate

Comment (abu-usama) [reply chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: No, not in the repo. The choco release workflow runs the update.ps1 which updates this file if a newer release of cli was created. The update.ps1 script also creates a .nupkg file that we then upload to choco. We actually don't need to update this info within the repo.

Comment (abu-usama) [reply chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1:9]: We could add an additional step to the workflow, that creates a PR for update changes.

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**